Moblin/Various/Model.swift

```diff
@@ -4955,13 +4970,37 @@ extension Model {
     }
 
     func updateRemoteControlAssistantStatus() {
-        guard showingRemoteControl, remoteControlAssistant?.isConnected() == true else {
+        guard showingRemoteControl || isWatchReachable(), remoteControlAssistant?.isConnected() == true else {
             return
         }
         remoteControlAssistant?.getStatus { general, topLeft, topRight in
             self.remoteControlGeneral = general
             self.remoteControlTopLeft = topLeft
             self.remoteControlTopRight = topRight
+            var thermalState: ProcessInfo.ThermalState
+            switch self.remoteControlGeneral?.flame {
+            case .white:
+                thermalState = .fair
+            case .yellow:
+                thermalState = .serious
+            case .red:
+                thermalState = .critical
+            case .none:
+                thermalState = .critical
+            }
+            self.sendThermalStateToWatch(thermalState: thermalState)
+            if let recordingMessage = topRight.recording?.message {
+                self.sendRecordingLengthToWatch(recordingLength: recordingMessage)
+            }
+            self.sendIsRecordingToWatch(isRecording: topRight.recording?.message != nil)
+            if let bitrateMessage = topRight.bitrate?.message {
+                self.sendSpeedAndTotalToWatch(speedAndTotal: bitrateMessage)
+            }
+            self.sendIsLiveToWatch(isLive: topRight.bitrate?.message != nil)
```

Comment:
Why it `bitrate` used to check if live? Should we add the live state to the remote control protocol?

Reply:
done